misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenUtils.java

```diff
@@ -256,49 +325,41 @@ public static void createDirectory(String path) throws BindgenException {
     }
 
     public static boolean isPublicField(Field field) {
-
         int modifiers = field.getModifiers();
         return Modifier.isPublic(modifiers);
     }
 
     public static boolean isPublicMethod(Method method) {
-
         int modifiers = method.getModifiers();
         return Modifier.isPublic(modifiers);
     }
 
     public static boolean isPublicClass(Class javaClass) {
-
         int modifiers = javaClass.getModifiers();
         return Modifier.isPublic(modifiers);
     }
 
     public static boolean isStaticField(Field field) {
-
         int modifiers = field.getModifiers();
         return Modifier.isStatic(modifiers);
     }
 
     public static boolean isStaticMethod(Method method) {
-
         int modifiers = method.getModifiers();
         return Modifier.isStatic(modifiers);
     }
 
     public static boolean isFinalField(Field field) {
-
         int modifiers = field.getModifiers();
         return Modifier.isFinal(modifiers);
     }
 
     public static boolean isAbstractClass(Class javaClass) {
-
         int modifiers = javaClass.getModifiers();
         return Modifier.isAbstract(modifiers);
     }
 
     public static String getModuleName(String jarPath) {
```

Comment:
Have we used this method?

Reply:
No, ATM support for generating bindings for complete jar files is not provided, but we intend on providing that in the future. This method could be used when implementing that feature in order to figure out the name of the module and that is why it is not removed even though it is not used.